src/neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -28,13 +28,13 @@ public class Transaction : IEquatable<Transaction>, IInventory, IInteroperable
 
         private byte version;
         private uint nonce;
-        private UInt160 sender;
         private long sysfee;
         private long netfee;
         private uint validUntilBlock;
         private TransactionAttribute[] attributes;
         private byte[] script;
         private Witness[] witnesses;
+        private Signers _signers;
```

Comment:
Signer[] ?  It seems `Signers` is not very necessary.

Reply:
Agree.

Reply:
Done